docs/tutorial/fast_data/fast_data_tutorial.mdx

```diff
@@ -316,14 +369,42 @@ Here is the response for the Order Projection after we created an order with id
 ```
 
 ## Single View
+
 The Single View is an architectural object that it's used to store the aggregate data from the Projections. The goal of the Single View is to store the data already aggregated and ready to use from the Projection.
 A good approach is to store the data in Single Views dedicated to each channel and each use. For example, if inside the client we use the data from the registry and the order in the same interface, we can create a Single View called "registry_and_order" and store all the useful data in this Single View.
 
-In this tutorial our Single View is called "order-history" and we will create an object schema with the registry information and the orders information linked to the registry.
-So, for this tutorial we will create one Single View called "order-history".
+In this tutorial, our Single View is called _order-history_ and we will create an object schema with the registry information and the orders information linked to the registry.
+
+### Microservice Single View Creator Configuration
+
+First of all, we need to create a microservice that we will attach to the Single View that we're going to create. This microservice is called "Single View Creator", and it's available in the microservice Marketplace.
+![Marketplace and Single View Creator](img/23-marketplace-single-view-low-code.png)
+
+Here we have two different plugins: the first one is for the Single View Creator, and the second one is for the Single View Creator Low Code. 
+We will go for the latter since we want to leverage the usage of ConfigMaps for a quicker configuration of the microservice. We suggest naming it _single-view-creator-delivery_.
 
-### Order History 
-Inside our Single View, each document is an object that contains the registry information about the user, and the information about the orders associated to the user.
+Now we need to configure all the parameters of the microservice, here is an example of the fields that we need to configure:
+
+- **PROJECTIONS_CHANGES_COLLECTION**: it is the name of the collection that includes the Projection Changes generated by the Real-Time Updater, the two must share the same name (we previously suggested _fd-pc-delivery_).
```

Comment:
`the two must share the same name` what does `the two` refer to? The two projections?

Reply:
let me know if you like it more this way